src/components/Toggle/Toggle.css

```diff
@@ -4,6 +4,20 @@
   z-index: var(--toggle-span-z-index, 1);
 }
 
+.toggle-span[category='neutral']:hover {
```

Comment:
before it was getting a button, and it is weird that we have a button with a `toggle-span` class. Can you please check it out?